### PR TITLE
Kinseyj: How to make an Iced Honey Lavender Shaken Espresso

### PR DESCRIPTION
**Pull request by jckinsey01**
## Kinseyj: p-make-espresso-honey-vanilla

**Completed Work**

- Created new procedure: `p-make-iced-espresso-honey-lavender.md`
- Procedure's Goal: To teach those familiar with making espresso drinks a new recipe for an iced honey lavender shaken espresso.

## User Scenario

 A graduate student who is spending too much money going to a chain cafe for coffee in the morning. They want to still be able to have caffeine in their day, and want to enjoy it. They have a nespresso machine/ nespresso machine dupe/ espresso machine. They want to make a delicious morning coffee that’ll get them their caffeine fix while still being fun, enjoyable and cheap. 

## Notable Review Requests

Below our some guiding questions and concerns, regarding the above work. Reviewers can use this information to start their review process, but need not limit their feedback to these items alone.

* I plan to add images I just find that process confusing and needed some time to figure it out
* Should I format my alerts differently?
* Does the order I have my sections in make sense?
* [What I have so far](https://github.com/ENG517/Procedure/commit/66d72dbc2767aefb72f40952f7e906fec13c931e)

### CODE REVIEW

File: kinseyj/p-make-iced-espresso-honey-lavender.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# How to Make an Iced Honey Lavender Shaken Espresso
+
+Want a new espresso recipe to make at home that's cheaper than a certain coffee chain and just as good, if not better? Here is a recipe for a delicious iced shaken espresso recipe with an amazing flavor combination.
```
Comment (lily-soetebier) [inline kinseyj/p-make-iced-espresso-honey-lavender.md:3]: I really like this introduction! It appeals well to the target audience you outlined, and provides enough context to the situation without being too wordy.

File: kinseyj/p-make-iced-espresso-honey-lavender.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Utentils  and Supplies To Get Started
+
+* An espresso maker of some kind
```
Comment (lily-soetebier) [inline kinseyj/p-make-iced-espresso-honey-lavender.md:7]: It may be useful to include common brands/names of espresso machines/making methods (i.e. nespresso, moka pot, instant espresso)

**Review by lily-soetebier — COMMENTED** [review]
This is a really strong draft of this procedure! I have left a few in-line comments about specific details that are really strong, and moments where you need a little more specificity. Below is some higher level feedback to get you started!
## Staging
* Your introduction is strong and well suited for your audience
*  The pre-requisites you include are comprehensive, but not overwhelming. One organizational note I have is maybe adding in a third subcategory of equipment (for the espresso maker and freezer) that separates them from the utensils. I feel like these are the most likely for your potential user to not have, and thus seeing this from the get go can help them decide if your recipe is something they can see themselves making

##Coaching
* The number and level of steps you have works really well
* It might be helpful to break up the steps into brewing/ flavoring, and assembling I think this break comes naturally with the 5-10 minute waiting period, but it could be helpful to denote that visually with subheadings

## Alerting
* Your alerts don't really stand out to me, maybe use italics or bolding to help them really pop-out from the rest of the procedure

Overall this was a really good draft!

**Comment by kb-boop** [comment]
## Staging 
Could you add an estimated amount of time to make? This might be helpful for a time-crunched student

This looks delicious!
